public/app/features/canvas/runtime/element.tsx

```diff
@@ -71,7 +71,7 @@ export class ElementState implements LayerElement {
   }
 
   /** Use the configured options to update CSS style properties directly on the wrapper div **/
-  applyLayoutStylesToDiv() {
+  applyLayoutStylesToDiv(disable?: boolean) {
```

Comment:
Maybe we can be a bit more specific with this parameter as `disable` is pretty broad, maybe `disablePointerEvents`? (across all uses of this (including in the `scene` file))